include/rmm/mr/device/polymorphic_allocator.hpp

```diff
@@ -302,6 +302,9 @@ bool operator!=(stream_allocator_adaptor<A> const& lhs, stream_allocator_adaptor
  * @return A `stream_allocator_adaptor` wrapping `allocator` and `s`
  */
 template <typename Allocator>
+[[deprecated(
+  "make_stream_allocator_adaptor is deprecated. Use the stream_allocator_adaptor constructor "
+  "instead.")]]
```

Comment:
*issue*: Please update the docstring of `stream_allocator_adaptor` to use the preferred approach in its example.

Reply:
Yes, line 169 above. Also, the same example exists in RMM's `README.md`, please update there as well.

Reply:
Done.